diff -aurN gtk-sharp-1.0.10.orig/configure.in gtk-sharp-1.0.10/configure.in
--- gtk-sharp-1.0.10.orig/configure.in	2005-05-14 19:05:17.000000000 +0200
+++ gtk-sharp-1.0.10/configure.in	2005-07-12 15:49:05.000000000 +0200
@@ -64,7 +64,7 @@
 PKG_CHECK_MODULES(MONO_DEPENDENCY, mono >= $MONO_REQUIRED_VERSION, has_mono=true, has_mono=false)
 
 if test "x$has_mono" = "xtrue"; then
-GACUTIL_FLAGS='/package $(PACKAGE) /gacdir $(DESTDIR)$(prefix)/lib'
+GACUTIL_FLAGS='/package $(PACKAGE) /root $(DESTDIR)$(libdir)'
 GENERATED_SOURCES=generated/*.cs
 AC_PATH_PROG(RUNTIME, mono, no)
 AC_PATH_PROG(CSC, mcs, no)
