--- gtk-sharp-1.0.8/configure.in.orig	2005-03-14 23:30:14.000000000 +0100
+++ gtk-sharp-1.0.8/configure.in	2005-05-15 13:39:27.000000000 +0200
@@ -157,7 +157,7 @@
 PKG_CHECK_MODULES(GTKHTML_DEPENDENCIES, libgtkhtml-3.6 >= $GTKHTML_REQUIRED_VERSION, enable_gtkhtml=yes, enable_gtkhtml=no)
 if test "x$enable_gtkhtml" = "xyes"; then
 GTKHTML_VERSION=3.6
-GTKHTML_SOVERSION=18
+GTKHTML_SOVERSION=$GTKHTMLSOVERSION
 else
 GTKHTML_REQUIRED_VERSION=3.2.0
 PKG_CHECK_MODULES(GTKHTML_DEPENDENCIES, libgtkhtml-3.1 >= $GTKHTML_REQUIRED_VERSION, enable_gtkhtml=yes, enable_gtkhtml=no)
